index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="ku">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="her yêke û şitllêk biçênîn, Daristanêk pêk dênîn">
+    <title>Daristan</title>
+    <link rel="stylesheet" href="css/master.css">
+</head>
+<body>
+    
```

Comment:
wa baştir bû şitêkiman danaba

Reply:
@ferbun-gelawej  bellam bedaxewe nusirawem we gîr nekewto